### ISSUE TITLE
Cannot read properties of undefined error? 

### ISSUE THREAD

**Issue by TheFreshPrins** [OrangeJedi/Aerial#116]
Sometime during the day I get the following error: https://photos.app.goo.gl/UXLZLb7JzqTYkt1R6

Any solution for this problem? 

Some additional information;
*Clean windows installation without a previous Aerial install
*The screen looks "stuck" and you can't click on anything untill you press the Win key
*One monitor setup

Thanks!

**Comment by OrangeJedi** [comment]
Sorry to hear that Aerial isn't working. How often have you gotten this error? Is it when the screensaver is just running on its own, or after you try and interact with it? Does it crash the screensaver, or just pop up with the error?

It looks like the code that is checking to see if the window is closed isn't working. I'll take a closer look soon and see why that may be the case.

**Comment by TheFreshPrins** [comment]
> Sorry to hear that Aerial isn't working. How often have you gotten this error? Is it when the screensaver is just running on its own, or after you try and interact with it? Does it crash the screensaver, or just pop up with the error?
> 
> It looks like the code that is checking to see if the window is closed isn't working. I'll take a closer look soon and see why that may be the case.

Hey there, thanks for the quick reply. The error causes to crash the screensaver. It only happens on its own. I didn't configure anything yet.

It didn't happen the last few days to be honest. Don't know why.

I tried adding some custom video's like in V0.5.4, but they don't show up after converting them in VLC to MPEG4 1080P for TV/Device. Did you rewrite the custom video section of the application with some changes?

Thanks in advance!

**Comment by OrangeJedi** [comment]
Ok, thanks for the extra info. When I get the time I'll see if I can find anything that caused the initial error with the windows close. Also, none of the code for custom videos changed from v0.5.4, but I also wasn't able to get it working. I will take a look at that code too.

**Comment by OrangeJedi** [comment]
I ended up having some time tonight to go through the code. I wasn't able to pinpoint for certain why the original `read proprieties of undefined` happened, but I added some code that should prevent it for happening again to the v1.1.0 branch/release. I also found the problem with the custom videos, while the code didn't change, a few libraries were updated that caused some runtime errors to occur. I was able to fix those and get custom videos working again in v1.1.0.

Thanks for bringing this up! I'm glad that these two things were able to be fixed.
